### PR TITLE
xcode-test-bundle-builder 0.0.3

### PR DESCRIPTION
**Pull request by otusweb**
Follow up to pull request #679

Remove the duplicate version

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/xcode-test-bundle-builder/0.0.3/step.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+title: Build xcode test runner for remote testing service
+summary: Build your XCTest or XCUITest runner to upload to remote testing service
+description: Build your XCTest or XCUITest bundle to upload to remote testing service
```
Comment (viktorbenei) [inline steps/xcode-test-bundle-builder/0.0.3/step.yml]: Circle CI doesn't have a remote testing service AFAIK - did you mean another service?

Comment (viktorbenei) [inline steps/xcode-test-bundle-builder/0.0.3/step.yml]: managed to mark the wrong line.. - should be https://github.com/bitrise-io/bitrise-steplib/pull/694/files#diff-5517e5d4663d90868de4ed46bff4a32fR4

**Comment by viktorbenei** [comment]
Can you please comment on my the question in the review comment?

File: steps/xcode-test-bundle-builder/0.0.3/step.yml (hunk 1 above)
Comment (otusweb) [inline steps/xcode-test-bundle-builder/0.0.3/step.yml]: You're right, they don't. I'll go back and edit.

Comment (viktorbenei) [inline steps/xcode-test-bundle-builder/0.0.3/step.yml]: Thanks!

Everything else seems to be perfect, so just edit, create new version and share it, and will be merged in no time ;)

**Pull request closed without merging**